### DISCUSSION TITLE
No update for Chrome and Edge

### DISCUSSION THREAD

**Discussion by maicol07** [darkreader/darkreader#8880] [General]
Hi,
do you have plans to update the Chromium extension on Microsoft Store and Chrome Web Store?
Currently we are missing 4.9.49 and 4.9.50
Thanks

**Comment by Gusted** [comment]
4.9.49 never got released, due to some errors.
4.9.50 was released to Firefox to fix a bug in Firefox Nightly. https://github.com/darkreader/darkreader/issues/8789

There's no benefit to release it to chrome/edge.

**Reply by maicol07** [reply]
The changelog also mentions some fixes to websites...

**Reply by Gusted** [reply]
The fixes to the site can also be fetched on older versions, see https://github.com/darkreader/darkreader#site-fixes
